### ISSUE TITLE
HTML-file causes build warning in ASDF

### ISSUE THREAD

**Issue by rpgoldman** [michaelw/cl-dot#15]
Is there some reason we need to have this in the system definition?

``` common-lisp
(:module "docs"
            :components
            ((:html-file "cl-dot")))
```

It doesn't seem to serve any constructive purpose, and it causes the following in ASDF:

```
Warning: compiling #<HTML-FILE "cl-dot" "docs" "cl-dot"> completed without its input file
         #P"/Users/rpg/lisp/vendor/cl-dot/docs/cl-dot.html"
Warning: loading #<HTML-FILE "cl-dot" "docs" "cl-dot"> completed without its input file
         #P"/Users/rpg/lisp/vendor/cl-dot/docs/cl-dot.html"
```

I suppose one could add a build rule that would trigger "make" to create that file, but then that would cause the build to fail on any machine that didn't have texinfo installed.

It occurs to me that this may be a bug in ASDF...

**Comment by rpgoldman** [comment]
Wait: not a bug in ASDF.  That file does not exist -- because it's constructed by the makefile -- so ASDF is correct to complain that it is missing.

**Comment by marcoheisig** [comment]
Fixed in 73dfbb6e015a28ebed873266e4e8190e509b43de.
